src/com/xilinx/rapidwright/rwroute/RWRoute.java

```diff
@@ -1423,10 +1423,25 @@ protected void setPIPsOfNets() {
         for (Entry<Net,NetWrapper> e : nets.entrySet()) {
             NetWrapper netWrapper = e.getValue();
             Net net = netWrapper.getNet();
+
```

Comment:
Same question as above, do we want to exclude GND/VCC nets?

Reply:
Static/global nets are dealt with by `GlobalSignalRouter` and should not be processed here. Added an assert just to be clear: 8317f6d